README.md

```diff
@@ -24,13 +24,9 @@ Inspired by Doomspork's [Elixir companies][elixir-companies] list.
   Pivotal Tracker provide a project management tool for agile software
   development teams.
 
+#### Other
 
-#### Sustainability Technology
-
-* [imby.bio](http://imby.bio/)
-  ([Gitlab](https://gitlab.com/imby.bio), [GitHub](https://github.com/imbybio/)) -
-  imby.bio helps individuals bring nature back in to their urban environment.
-
+* [Hearken] (https://www.wearehearken.com) ([Github](https://github.com/wearehearken/)) - Hearken provides process consulting and supporting technology for media organizations to better engage with their audience.
```

Comment:
Seems you removed another company here.

Would it be possible to have a more specific section than "Other"?